lib/datadog/appsec/contrib/devise/patcher.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require_relative '../patcher'
+require_relative 'resource'
+require_relative 'ext'
+require_relative '../../../kit/appsec/events'
+
+module Datadog
+  module AppSec
+    module Contrib
+      module Devise
+        AUTOMATED_USER_EVENT_TAGGING_BLOCK = proc do |trace, event|
+          trace.set_tag("_dd.appsec.events.#{event}.auto.mode", AppSec.settings.automated_track_user_events)
+        end
+
+        private_constant :AUTOMATED_USER_EVENT_TAGGING_BLOCK
+
+        # Hook in devise validate method
+        module AuthenticablePatch
```

Comment:
I think there's a typo?

```suggestion
        module AuthenticatablePatch
```